pkg/api/types.go

```diff
@@ -431,6 +439,7 @@ type ContainerState struct {
 }
 
 type ContainerStatus struct {
+	Ready bool `json:"ready"`
```

Comment:
I'm ok with this for now, though I imagine we'll need more info later, as in PodCondition or ContainerStateTerminated. However, organizationally, please put it below State.